woocommerce/woocommerce-ios
# My Store Analytics: Various beta feedback items

Closes: #12305


## Description


This PR adds the following changes for Custom Range tab:
1. Keep the date range label tappable all the time, whether chart is selected or not
2. Update icon to be pencil and for it to be trailing
3. Replace hyphen between date range to en-dash


## Testing instructions

1. Create a Custom Range tab if you haven't
2. Ensure the date range label is shown with the pencil icon to the end of it,
3. Select a value in the chart,
4. Ensure the date range label is unchanged from before


## Screenshots


| | Before | After |
|--------|--------|--------|
| Icon change, and hyphen changed to en-dash | ![Simulator Screenshot - iPhone 15 - 2024-03-19 at 15 17 29](https://github.com/woocommerce/woocommerce-ios/assets/266376/f28b5dbd-2915-4aba-a580-a0714a1e1925) | ![Simulator Screenshot - iPhone 15 - 2024-03-19 at 15 19 13](https://github.com/woocommerce/woocommerce-ios/assets/266376/72ebf8c3-830c-4ef7-afcd-1c3e679ab505) |
| Label still shown on value selection | ![Simulator Screenshot - iPhone 15 - 2024-03-19 at 15 17 25](https://github.com/woocommerce/woocommerce-ios/assets/266376/14df62cf-00dd-4981-8e8f-d6161d32f7e1) | ![Simulator Screenshot - iPhone 15 - 2024-03-19 at 15 14 42](https://github.com/woocommerce/woocommerce-ios/assets/266376/ed4d7ea7-2972-4251-8605-471fc6a70779) |
